great_expectations/metrics/column_pair_values/in_set.py

```diff
@@ -0,0 +1,15 @@
+from typing import Any
+
+from great_expectations.metrics.domain import ColumnPairValues
+from great_expectations.metrics.metric import Metric
+from great_expectations.metrics.metric_results import MetricResult
+
+
+class ColumnPairValuesInSetUnexpectedCountResult(MetricResult[int]): ...
+
+
+class ColumnPairValuesInSetUnexpectedCount(
+    Metric[ColumnPairValuesInSetUnexpectedCountResult], ColumnPairValues
+):
+    name = "column_pair_values.in_set.unexpected_count"
```

Comment:
Can you add the string `unexpected_count` to the enum `MetrixNameSuffix` and use it here? We have one (`condition`) that is a special case and I have been trying to keep an inventory of them just in case they have some meaning. I think they get used more generally in the legacy system metric "provider" superclasses.

Reply:
ok - i've updated this, but i don't totally understand the utility. imo, this makes the code significantly less readable, so if there isn't an immediate benefit here, i suggest we just revert to plain strings.